File: onebot/src/main/java/cn/evole/onebot/sdk/response/group/EssenceMsgResp.java
package cn.evole.onebot.sdk.response.group;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

/**
 * @author cnlimiter
 */
@Data
public class EssenceMsgResp {

    @SerializedName("sender_id")
    public long senderId;

    @SerializedName("sender_nick")
    public String senderNick;

    @SerializedName("sender_time")
    public long senderTime;

    @SerializedName("operator_id")
    public long operatorId;

    @SerializedName("operator_nick")
    public String operatorNick;

    @SerializedName("operator_time")
    public String operatorTime;

    @SerializedName("message_id")
    public int messageId;

}

File: onebot/src/main/kotlin/sdk/response/group/EssenceMsgResp.kt
package cn.evolvefield.onebot.sdk.response.group

import com.google.gson.annotations.SerializedName

class EssenceMsgResp {
    @SerializedName("sender_id")
    var senderId = 0L
    @SerializedName("sender_nick")
    var senderNick = ""
    @SerializedName("sender_time")
    var senderTime = 0L
    @SerializedName("operator_id")
    var operatorId = 0L
    @SerializedName("operator_nick")
    var operatorNick = ""
    @SerializedName("operator_time")
    var operatorTime = ""
    @SerializedName("message_id")
    var messageId = 0
}
